build.gradle.kts

```diff
@@ -3,13 +3,20 @@
  */
 
 @file:Suppress("SpellCheckingInspection")
+
+import de.fayard.BuildSrcVersionsExtension
+
 plugins {
     id("de.fayard.buildSrcVersions").version("0.6.3")
     id ("com.gradle.build-scan").version("2.4.2") 
 }
 
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
+// Libs.kotlinVersion comes from gradle.properties. Keep in sync with buildSrc/build.gradle.kts
+Libs.kotlinVersion = findProperty("version.kotlin-stdlib-common") as String
```

Comment:
Why not just use `version.kotlin`?

Reply:
because it's not generated by `:refreshVersions` and would be overwritten next time we run the task